src/Xamarin.Android.Build.Tasks/Tasks/AndroidToolTask.cs

```diff
@@ -99,7 +99,7 @@ protected override void LogEventsFromTextOutput (string singleLine, MessageImpor
 )?
 ( # optional warning|error:
  \s*
- (?<level>(warning|error)[^:]*)\s*
+ (?<level>(warning|error|Error|ERROR)[^:]*)\s*
```

Comment:
Maybe we could just use `RegexOptions.IgnoreCase`, and it would also handle if they decide to use `Warning` or `WARNING`?